### PR TITLE
configure pulp proxy integration

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: roles/pulp/tasks/main.yaml [hunk 1]
```diff
@@ -37,3 +37,9 @@
     host: "{{ ansible_hostname }}"
     port: 8080
     timeout: 300
+
+# Only needed until we have cert auth configured
```
Comment (evgeni) [inline roles/pulp/tasks/main.yaml:41]: Split out to #26

File: roles/foreman/tasks/main.yaml [hunk 2]
```diff
@@ -113,3 +113,11 @@
   register: foreman_tasks_status
   when:
     - "'katello' in foreman_status.json['results']"
+
+- name: Configure Foreman Proxy
+  theforeman.foreman.smart_proxy:
```
Comment (ehelms) [inline roles/foreman/tasks/main.yaml:118]: That's fun!

**Review by ehelms — APPROVED** [review]
After rebase

File: roles/foreman/tasks/main.yaml (hunk 2 above)
Comment (evgeni) [reply roles/foreman/tasks/main.yaml:118]: You mean dogfooding?

**Pull request merged**